contracts/ante/AnteDumpTest.sol

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.8.0;
+
+import "../interfaces/IERC20.sol";
+import "../AnteTest.sol";
+
+struct WalletTest {
+    address wallet;
+    address token;
+    uint256 amount;
+    uint256 timeRegistered;
+    uint256 timeValid;
+    uint256 expiry;
+}
+
+/// @title  AnteDumpTest
+/// @notice Ensures a wallet doens't dump x amount of tokens over y time
+contract AnteDumpTest is AnteTest("Ensure a set of wallets doesn't dump their associated tokens") {
+    
+    WalletTest[] private walletTests;
+    uint8 public immutable thresholdPercent;
+
+    address public immutable admin;
+    address public immutable owner;
+
+    modifier onlyAdmin() {
+        require(msg.sender == admin || msg.sender == owner, "ANTE: Must be an admin or owner");
+        _;
+    }
+
+    /// @notice Initializes the test
+    /// @param _tokenAddress A list of adddresses for tokens
+    /// @param _monitorWallet a list of wallets to monitor
+    /// @param _thresholdPercent the percentage of tokens that must be owned by a wallet
+    /// @param _timeValid the time that a wallet condition is valid
+    /// @dev When passing in a list, each wallet address will correspond to the token at the same index
+    /// @dev Eg [USDC, DAI] | [WALLET1, WALLET2] - WALLET1 will own USDC and WALLET2 will own DAI
+    constructor(address[] memory  _tokenAddress, address[] memory _monitorWallet, uint8 _thresholdPercent, uint256 _timeValid, address _admin) {
+
+        protocolName = "Ante";
+        testedContracts = _tokenAddress;
+
+        thresholdPercent = _thresholdPercent;
+
+        admin = _admin;
+        owner = msg.sender;
+
+        for(uint256 i = 0; i < _monitorWallet.length; i++) {
+            IERC20 token = IERC20(_tokenAddress[i]);
+
+            WalletTest memory walletTest;
+
+            walletTest.wallet = _monitorWallet[i];
+            walletTest.token = _tokenAddress[i];
+            walletTest.amount = token.balanceOf(_monitorWallet[i]);
+            walletTest.timeRegistered = block.timestamp;
+            walletTest.timeValid = _timeValid;
+            walletTest.expiry = block.timestamp + _timeValid;
+
+            walletTests.push(walletTest);
+        }
+    }
+
```

Comment:
I think we should add a "changeAdmin" function, where an owner or admin can replace the admin address with a new one.